app/views/articles/_form.html.erb

```diff
@@ -15,6 +15,14 @@
     <% end %>
   </div>
 
+  <div>
+    <%= form.label :status %><br>
+    <%= form.select :status, ['public', 'private', 'archived'], selected: 'public' %>
```

Comment:
Could we use the same variable VALID_STATUSES?

Reply:
The VALID_STATUSES variable cannot be used because it's not a class variable and for this case, what is done is to give the options of the type of article to be published.